logic/staff/staff.go

```
@@ -0,0 +1,208 @@
+		}
+
+		account.StaffId = &staff.Id
+	}
+
+	// 创建账号
+	result := gplus.Insert(&account)
+	if result.Error != nil {
+		return errors.New("创建账号失败")
+	}
+
+	go l.sendCreateMessage(ctx, account, staff)
```

golangci-lint (1.62.2)

Error return value of `l.sendCreateMessage` is not checked

(errcheck)